[docs](function) support aggregrate function regr_slope regr_intercept

Yoruet opened the pull request:
# Versions 

- [x] dev
- [x] 3.0
- [ ] 2.1
- [ ] 2.0

# Languages

- [x] Chinese
- [x] English

https://github.com/apache/doris/pull/41240

KassieZ commented:
hi, please update dev version as well

KassieZ commented:
For SQL reference, please fix the title format, you could refer to this sample :

```markdown
---
{
    "title": "FUNCTION_NAME",
    "language": "en"
}
---

<!--
Licensed to the Apache Software Foundation (ASF) under one
or more contributor license agreements.  See the NOTICE file
distributed with this work for additional information
regarding copyright ownership.  The ASF licenses this file
to you under the Apache License, Version 2.0 (the
"License"); you may not use this file except in compliance
with the License.  You may obtain a copy of the License at

  http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing,
software distributed under the License is distributed on an
"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
KIND, either express or implied.  See the License for the
specific language governing permissions and limitations
under the License.
-->

## Description

`FUNCTION_NAME` is used to XXXXXXX for XXXXX。(A brief description of the function and its purpose. Describe the use cases and scenarios where this function can be beneficial.)

Since Doris vXXX is supported XXXX。(Optional)

## Syntax

```sql
FunctionName(param1, param2, ...)
```
## Paremeters

- **param1 (DataType1)**: Brief description of the parameter and its role in the function.

- **param2 (DataType2)**: Same.
- ...

## Returned values

Returned data type: `ReturnType` 

- If XXXcase1 then returned XXXX value1；

- If XXXXcase2 then returned XXXX value2；

## Examples

```sql
-- Example 1: Basic Usage
SELECT FunctionName('example_value') FROM dummy_table;  
  
-- Example 2: Usage in query
SELECT column1, FunctionName(column2) AS result_column FROM example_table;  
  
-- Example 3: Advanced usage
SELECT column1, FunctionName(OTHER_FUNCTION(column2)) AS combined_result FROM example_table;
```

## Usage notes

- List any known limitations or restrictions on the usage of function. 

- This could include data size limits, unsupported data types, or specific behavior that may not be intuitive.

## Related commands

[FUNCTION_NAME_2](../../sql-manual/sql-functions/XXXXXXX)、[FUNCTION_NAME_3](../../sql-manual/sql-functions/XXXXXXX)、[More...](../../XXXX)

## References

For more details about XXXX, please refere to [XXXXX](../../XXXXX/XXXXX)
```

Yoruet commented:
> hi, please update dev version as well

Sorry, I'm not sure what this means. I have added doc in version-3.0.

Yoruet commented:
> hi, please update dev version as well

Sorry, I'm not sure what this means. I have added doc in version-3.0.

KassieZ commented:
> > hi, please update dev version as well
> 
> Sorry, I'm not sure what this means. I have added doc in version-3.0.

Please also update EN docs on the directory : doris-website/doris-website/docs and CN docs on the directory : /doris-website/i18n/zh-CN/docusaurus-plugin-content-docs/current

Yoruet commented:
> > > hi, please update dev version as well
> > 
> > 
> > Sorry, I'm not sure what this means. I have added doc in version-3.0.
> 
> Please also update EN docs on the directory : `doris-website/doris-website/docs` and CN docs on the directory : `/doris-website/i18n/zh-CN/docusaurus-plugin-content-docs/current`

I have added docs:`docs/sql-manual/sql-functions/aggregate-functions/regr_intercept.md`  , 'docs/sql-manual/sql-functions/aggregate-functions/regr_slope.md' , 'i18n/zh-CN/docusaurus-plugin-content-docs/current/sql-manual/sql-functions/aggregate-functions/regr_intercept.md', 'i18n/zh-CN/docusaurus-plugin-content-docs/current/sql-manual/sql-functions/aggregate-functions/regr_slope.md'

Yoruet commented:
> > > hi, please update dev version as well
> > 
> > 
> > Sorry, I'm not sure what this means. I have added doc in version-3.0.
> 
> Please also update EN docs on the directory : `doris-website/doris-website/docs` and CN docs on the directory : `/doris-website/i18n/zh-CN/docusaurus-plugin-content-docs/current`

I thought that I already satisfied the requirement

KassieZ commented:
Thank you for the contribution, please also fix the conflicts.

zclllyybb commented:
has some wrong. replaced by https://github.com/apache/doris-website/pull/2443
